pkgs/build-support/skaware/build-skaware-man-pages.nix

```diff
@@ -0,0 +1,51 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+{
+  # : string
+  pname
+  # : string
+, version
+  # : string
+, sha256
+  # : list (int | string)
+, sections
+  # : string
+, description
+  # : list Maintainer
+, maintainers
+  # : license
+, license ? lib.licenses.isc
+  # : string
+, owner ? "flexibeast"
+  # : string
+, rev ? "v${version}"
+}:
+
+let
+  manDir = "${placeholder "out"}/share/man";
+
+  src = fetchFromGitHub {
+    inherit owner rev sha256;
+    repo = pname;
+  };
+in
+
+stdenv.mkDerivation {
+  inherit pname version src;
+
+  makeFlags = [
+    "MANPATH=${manDir}"
+  ];
+
+  dontBuild = true;
+
+  preInstall = lib.concatMapStringsSep "\n"
+    (section: ''mkdir -p "${manDir}/man${builtins.toString section}"'')
```

Comment:
```suggestion
    (section: "mkdir -p \"${manDir}/man${builtins.toString section}\"")
```

Reply:
What's the problem here? I thought you found escape sequences in double quoted strings unreadable?

Reply:
A while ago there was a PR that removed '' '' from single line strings. Just following that.

Reply:
Makes sense. Kinda slipped my mind.